Makefile

```diff
@@ -22,8 +22,14 @@ test: check-env build
 	cd ${dir}
 	npm test
 
+e2e-test: check-env build
+	cd ${dir}
+	npm run system-test
+
 lint:
+	npm install
 	cd ${dir}
+	npm install
```

Comment:
suggestion: Why two `npm install`? A comment here would be useful to understand what's happening.

Reply:
I copied what was going on the `test` command. Apparently we have to first install the root package which contains all the testing dependencies, and then the actual code sample package. I'll add some comments on both.